components/Cards/index.js

```diff
@@ -17,3 +17,70 @@
 // </div>
 //
 // Create a card for each of the articles and add the card to the DOM.
+const cardEntry = document.querySelector('.cards-container');
+
+axios.get('https://lambda-times-backend.herokuapp.com/articles')
+ .then(response => {
+  // console.log("articles", response);
+  response.data.articles;
+  response.data.articles.javascript.forEach(item => {
```

Comment:
Try to make this part a little more DRY. Look into these methods that allow you to iterate over objects: Object.values, Object.entries, or Object.keys. Here's a hint: you might want to use one of these methods on response.data.articles (ex: `Object.values(response.data.articles)` -- this should return to you an array that you can then loop over).

Here's the docs on Object.values: [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values]